src/main/kotlin/app/revanced/patches/interaction/EnableSeekbarTapping.kt

```diff
@@ -39,45 +37,21 @@ class EnableSeekbarTapping : Patch("enable-seekbar-tapping") {
             if (literal == 2) tapSeekMethods["O"] = it
         }
 
-        val implementation = cache.methodMap["enable-seekbar-tapping"].method.implementation!!
+        // replace map because we dont need the upper one anymore
+        map = cache.methodMap["enable-seekbar-tapping"]
+
+        val implementation = map.method.implementation!!
 
         // if tap-seeking is enabled, do not invoke the two methods below
         val pMethod = tapSeekMethods["P"]!!
         val oMethod = tapSeekMethods["O"]!!
+
         implementation.addInstructions(
             map.scanData.endIndex,
-            listOf(
-                BuilderInstruction35c(
-                    Opcode.INVOKE_VIRTUAL,
-                    0,
-                    3,
-                    0,
-                    0,
-                    0,
-                    0,
-                    ImmutableMethodReference(
-                        oMethod.definingClass,
-                        oMethod.name,
-                        setOf("I"),
-                        "V"
-                    )
-                ),
-                BuilderInstruction35c(
-                    Opcode.INVOKE_VIRTUAL,
-                    0,
-                    3,
-                    0,
-                    0,
-                    0,
-                    0,
-                    ImmutableMethodReference(
-                        pMethod.definingClass,
-                        pMethod.name,
-                        setOf("I"),
-                        "V"
-                    )
-                )
-            )
+            """
+               invoke-virtual { v12, v2 }, ${oMethod.definingClass}->${oMethod.name}(I)V
+               invoke-virtual { v12, v2 }, ${pMethod.definingClass}->${pMethod.name}(I)V
```

Comment:
That `v12` looks a little arbitrary and error-prone. Could you use an offset instead? Maybe we should add helper methods to get the register used at a instruction to get an offset that way?

Reply:
@oSumAtrIX ?

Reply:
IIRC two instructions afterwards, there are instructions referring to the same register, we can use them

Reply:
Fixed in 0924ca2ad30ae865dcc0fd484cb0da517e827352